modules/setup/src/main/Mappings.scala

```diff
@@ -14,7 +15,7 @@ object Mappings {
   val aiVariants = number.verifying(Config.aiVariants contains _)
   val variantWithVariants = number.verifying(Config.variantsWithVariants contains _)
   val variantWithFenAndVariants = number.verifying(Config.variantsWithFenAndVariants contains _)
-  val time = number.verifying(HookConfig validateTime _)
+  val time = of[Double].verifying(HookConfig validateTime _)
```

Comment:
@Unihedro I did change validateTime together with the mapping: https://github.com/ornicar/lila/pull/1206/files#diff-911a78120d0f30fa517d49690c679ab1R117